exec/java-exec/src/main/codegen/templates/DateIntervalFunctionTemplates/DateToCharFunctions.java

```diff
@@ -65,7 +65,7 @@ public void setup() {
         byte[] buf = new byte[right.end - right.start];
         right.buffer.getBytes(right.start, buf, 0, right.end - right.start);
         String input = new String(buf, com.google.common.base.Charsets.UTF_8);
-        format = org.joda.time.format.DateTimeFormat.forPattern(input);
+        format = org.joda.time.format.DateTimeFormat.forPattern(input).withLocale(java.util.Locale.ENGLISH);
```

Comment:
I don't think that it is the right solution. A table may contain a field with date strings which were created with the non-ENGLISH locale so for this case, the query will fail. 

We need to set locale to ENGLISH only for required unit tests.